### PR TITLE
[LOOP-4596] Scenario Organization

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4596

### CODE REVIEW

File: LoopKitUI/OnboardingUI.swift [hunk 1]
```diff
@@ -136,6 +136,17 @@ public enum StudyProduct: String, CaseIterable {
     case none
     case studyProduct1
     case studyProduct2
+    
+    public func filtered(scenarioURLs: [URL]) -> [URL] {
```
Comment (ps2) [inline LoopKitUI/OnboardingUI.swift]: Sorry I missed the earlier PR where `SudyProduct` ended up here, but LoopKit should be independent of special Tidepool cases; it should be restricted (as much as possible) to providing general purpose tools that could be used by other diabetes focused app.

**Review by ps2 — CHANGES_REQUESTED** [review]

File: LoopKitUI/OnboardingUI.swift (hunk 1 above)
Comment (nhamming) [reply LoopKitUI/OnboardingUI.swift]: Yeah, I gave @Camji55 the same feedback on that previous PR but after discussion it seemed this was the best place for it.

@ps2 do you have something in mind?

Comment (Camji55) [reply LoopKitUI/OnboardingUI.swift]: Made some changes here

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**